templates/tfengine/components/devops/main.tf

```diff
@@ -104,13 +104,25 @@ module "owners_group" {
     module.project
   ]
 }
+
+resource "time_sleep" "owners_wait" {
```

Comment:
You should add a comment above this resource explaining why it's needed.

Reply:
Done